packages/starlight/index.ts

```diff
@@ -66,15 +65,11 @@ export default function StarlightIntegration(
 				);
 
 				const integrations = pluginResult.integrations;
+				const useTranslations = pluginResult.useTranslations;
+				const absolutePathToLang = pluginResult.absolutePathToLang;
```

Comment:
Should we refactor this to this?
```suggestion
				const { integrations, useTranslations, absolutePathToLang } = pluginResult;
```